### DISCUSSION TITLE
Flutter Help.

### DISCUSSION THREAD

**Discussion by futuraflutter** [community/community#124055] [Programming Help]
### Body

I try to use build an apk/bundle in Flutter with sign in key, and it shows an error of stored key path issue even I put right path in key.properties file. Can you guys help me?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by MidhileshRaj** [comment]
Try to remove the single quote or double inverted coma while giving the path in key.properties file.
Or double verify the name of the key store file as same as repeated in the same path or something like that.

**Comment by ebndev** [comment]
Hi @futuraflutter & @MidhileshRaj, thanks for being a part of the GitHub Community!

While we appreciate you trying to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
